CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to lektorium
+
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+## What should I know before I get started?
+1. We treat everyone and each as a friend and with respect. Particularly: taking it personally, insulting, indignity by any reason, critics and compilation personal attributes, which receiver cannot change (such as sex, age, health condition, sexual orientation, etc) are not allowed. We keep close to each other, joke, fool around and have fun from interaction, but in the frame what's written above.
```

Comment:
in term (or terms) written above (no "what's", maybe that. but that can be omitted)